src/community/ogcapi/ogcapi-tiled-features/src/main/java/org/geoserver/ogcapi/features/tiled/TiledFeatureService.java

```diff
@@ -20,6 +19,7 @@
 import org.geoserver.wfs.WFSInfo;
 import org.geowebcache.GeoWebCacheException;
 import org.geowebcache.layer.TileLayer;
+import org.geowebcache.layer.meta.TileJSON;
```

Comment:
This feels like it should be split out - its vaguely related, but its really a different change.